trunner/runners/common.py

```diff
@@ -308,21 +310,46 @@ def go(self):
 class Runner:
     """Common interface for test runners"""
 
+    @abstractmethod
```

Comment:
You may use `@abstractmethod` only when the class is derived from `ABC`, see: https://docs.python.org/3/library/abc.html#abc.abstractmethod